skycultures/Book_of_Fixed_Stars_English/constellationsart.fab

```diff
@@ -0,0 +1,54 @@
+﻿001 UMi.png 465 430 11767 294 131 79822 173 223 70692

```

Comment:
Please change abbreviations as above, and add a newline at end.

Reply:
done